docs/api/classifier/threshold.rst

```diff
@@ -3,51 +3,6 @@
 Discrimination Threshold
 ========================
 
-.. caution:: This visualizer only works for *binary* classification.
```

Comment:
Why is all this documentation being removed?

Reply:
good question, perhaps I missed one in the massive merge conflict